web/admin/application/configs/klear/model/RoutingPatternGroups.yaml

```diff
@@ -41,8 +41,7 @@ production:
             fields:
               - name${lang}
               - description${lang}
-              - regExp
-            template: "%name${lang}% %description${lang}% (%regExp%)"
+            template: "%name${lang}% %description${lang}%"
```

Comment:
It looks like this regression was introduced on https://github.com/irontec/ivozprovider/commit/a902f4ab7f19e592b0085d48a6d9df0002309482
Should we just rename the field @cruzccl?

Reply:
I think the correct fix is Renaming the field like  https://github.com/irontec/ivozprovider/commit/3b407da21878b0d9058f774549f5f7d4775a04e5 does

Reply:
Now I understand better which was the real problem. Commit changed. Thanks.